tutorials/cp-apim-openconnectors-hubsopt-manage/cp-apim-openconnectors-hubsopt-manage.md

```diff
@@ -53,7 +53,7 @@ primary_tag: products>sap-api-management
 
     ![Test Connection](06-test-connection.png)
 
-    >If successful, the response code is **200**.
+    >If successful, the response code is **200** and you might see 404 error. Do not get confuse with it. Your system is up and running, you can proceed.
```

Comment:
```suggestion
    >If successful, you should get a **200** response code. However, you may get a **404** response code. Don't worry. The system is up and running and you can proceed.
```
Is this what you meant? I still don't understand. What if I get a "real" 404?

Reply:
Your text implies that the 404 response may come or not and that it is not causing a real problem. But what if the user is getting a 404 response that is causing a problem and is stopping him from finishing the tutorial? Maybe you can elaborate?